packages/web-components/fast-components/__test__/rollup.config.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-env node */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import path from "path";
+import commonJS from "rollup-plugin-commonjs";
+import resolve from "rollup-plugin-node-resolve";
+import typescript from "rollup-plugin-typescript2";
+
+const copy = require("../../../../build/copy");
+
+const srcDir = path.resolve(__dirname, "..", "src");
+const distDir = path.resolve(__dirname, "public");
+
+export default [
+    {
+        context: "this",
+        input: path.resolve(srcDir, "index-rollup.ts"),
+        onwarn(warning, warn) {
+            // The IIFE export doesn't have a namespace since component exports
```

Comment:
Nit: For a single comment that wraps into two sentences, i'd prefer to use `/* ... */` instead of `//` which is great for one-line comments.

Reply:
There are examples of both styles throughout this project so I'm going to leave this as-is. If this is a project preference lets get it implemented in a linter or prettier.